tests/integration/utils.py

```diff
@@ -195,6 +195,13 @@ def _safe_extract(tar: TarFile, path: str = ".", *, numeric_owner: bool = False)
     tar.extractall(path, numeric_owner=numeric_owner)
 
 
+def _update_test_data_if_needed(test_data_path: Path, json_data: Dict) -> None:
+    if os.environ.get("CACHI2_UPDATE_TEST_DATA", False):
```

Comment:
Environment variables are strings, which means:
```python
>>> os.environ["CACHI2_UPDATE_TEST_DATA"]
'false'
>>> bool(os.environ["CACHI2_UPDATE_TEST_DATA"])
True
```
We should probably check that the value of the variable is `'true'`